app/password_util.py

```diff
@@ -0,0 +1,10 @@
+import hashlib
+
+class PasswordUtil():
+    salt = 'valyria'
+
+    @classmethod
+    def hash_password(self, password):
```

Comment:
AD, this could be outside the class. You'd still import from a module, so it'd be something like
```
from app.password_util import hash_password

hash_password(stuff)```

or we could also just have

```
from app import password_util

password_util.hash_password
```